templates/lean_projects.html

```diff
@@ -0,0 +1,134 @@
+      in other developments.
+      At the bottom of this page,
+      you can see a summary of which projects are compatible with which Lean versions.
+      If two projects both support the same Lean version,
+      you can likely use them together.
+    </p>
+
+    <p>
+      To add a project to this list,
+      please see the directions at the
+      <a href="https://github.com/leanprover-contrib/leanprover-contrib">leanprover-contrib</a> page.
+    </p>
+    </div>
+  </div>
+
+  <div class="row row-cols-1 row-cols-md-2">
+    {% for project in projects %}
+      <div class="col mb-4">
+      <div class="card h-100" >
+        <div class="card-body">
+          <h5 class="card-title">{{ project.name }}</h5>
+        <h6 class="card-subtitle mb-2 text-muted">maintained by
+          {% for maintainer in project.maintainers %}
+            <a href="https://github.com/{{ maintainer }}">@{{ maintainer }}</a>
+          {% endfor %}
+        </h6>
+        <p class="card-text">{{ project.description }}</p>
+        </div>
+            <div class="card-footer text-right bg-white">
+              <small><a href="https://github.com/{{ project.organization }}/{{ project.name }}" class="card-link">View on GitHub</a> (★ {{ project.stars }})</small>
+            </div>
+      </div>
+      </div>
+    {% endfor %}
+    </div>
+
+
+
+
+  <div class="row mt-5">
+    <div class="col-12">
+      <h2>Projects by supported Lean versions</h2>
+    </div>
+
+    <div class="col-12">
+      <p>
+        If you want to import multiple projects in your own,
+        you should choose a Lean version that is supported
+        by each of these dependencies.
+      </p>
+      <ul>
+        <li>
+          A ✓ in the <code>x.y.z</code> column
+          means that the project has a <code>lean-x.y.z</code> branch,
+          all of its dependencies have <code>lean-x.y.z</code> branches,
+          and the project builds successfully using the most recent commits
+          on all of these <code>lean-x.y.z</code> branches.
+        </li>
+        <li>
+          A × in the <code>x.y.z</code> column means that the project has an <code>lean-x.y.z</code> branch
+          but it fails to build.
+        </li>
+        <li>
+          No mark in the <code>x.y.z</code> column means that the project
+          does not have an <code>x.y.z</code> branch.
+        </li>
+      </ul>
+    </div>
+
+    <div class="col-12">
+       <div id="proj_table"></div>
+    </div>
+  </div>
+
+  <div class="row mt-5">
+
+  </div>
+{% endblock %}
+
+{% block extrajs %}
+<script type="text/javascript" src="https://leanprover-contrib.github.io/leanprover-contrib/projects.js"></script>
+
+<script>
+
+var color = Chart.helpers.color;
+var blue = 'rgb(54, 162, 235)'
+
+colors = ['#377eb8',
+    '#ff0029',  '#66a61e', '#984ea3', '#00d2d5', '#ff7f00',
+    '#af8d00', '#7f80cd', '#b3e900', '#c42e60', '#a65628', '#f781bf',
+    '#8dd3c7', '#bebada', '#fb8072']
```

Comment:
This looks like an unfortunate copy-paste result. What do you plan to do with those variables?

Reply:
I copied too much from the bottom of the stats page!